src/Microsoft.Extensions.DiagnosticAdapter/project.json

```diff
@@ -17,22 +17,17 @@
     "xmlDoc": true
   },
   "frameworks": {
-    "net451": {
-      "frameworkAssemblies": {
-        "System.Runtime": { "type": "build" }
-      }
-    },
-    "netstandard1.3": {
+    "netstandard1.1": {
       "dependencies": {
         "System.ComponentModel": "4.0.1-*",
         "System.Collections.Concurrent": "4.0.12-*",
         "System.Reflection.Emit": "4.0.1-*",
         "System.Reflection.Emit.Lightweight": "4.0.1-*",
         "System.Reflection.Extensions": "4.0.1-*",
-        "System.Reflection.TypeExtensions": "4.1.0-*"
```

Comment:
Is this package of extension methods really the thing that's TFM'ing us? It seems wrong that this package has a dependency on 5.4

/cc @davidfowl